admin/README.md

```diff
@@ -1,61 +1,86 @@
+## Development setup
+
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app) using
+scripts package [@nlesc/react-scripts](https://github.com/NLeSC/create-react-app). Original documentation
+[here](https://github.com/NLeSC/create-react-app/blob/master/packages/react-scripts/template/README.md).
+
+### Prerequisites
+
+1.  Install node version `^14.0`, e.g. using `nvm` (Node Version Manager, see https://github.com/nvm-sh/nvm#install-script):
+    1.  `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.36.0/install.sh | bash`
+    1.  restart terminal
+    1.  run `nvm install 14`
+1.  (Optional) Install `jq` for processing JSON config files:
 
-# Prerequisites
+    ```shell
+    sudo apt install jq
+    ```
 
-- [github-auth](/auth-github) authentication service.
-- [backend](/backend) data API service.
+### Getting a local development build running
 
-# Installation
+1.  Bring up the Research Software Directory as normal, except for the admin interface:
 
-1.  Clone repository
-2.  Use node version `^12.0`, e.g. using `nvm` (see [below](#install-node-using-nvm)).
-3.  Install yarn, `npm install -g yarn`
-4.  Install dependencies: `yarn install`
-5.  To run tests: `yarn test`
-6.  Start app in dev server: `yarn start`
+    ```shell
+    cd research-software-directory
 
-## Install node using NVM
+    docker-compose build && \
+       docker-compose up -d && \
+       docker-compose stop admin && \
+       docker-compose logs --follow
+    ```
 
-See https://github.com/creationix/nvm#install-script
+1.  New terminal
+1.  Get the IP addresses for the `auth` and `backend` services (Note `rsd_default` may be different for you if you
+    changed `COMPOSE_PROJECT_NAME` in `.env`):
 
-1.  curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | bash
-2.  restart terminal
-3.  run `nvm install 8`
+    ```shell
+    # where is auth running?
+    docker inspect $(docker-compose ps -q auth) | \
+       jq .[0].NetworkSettings.Networks.rsd_default.IPAddress
 
-## Settings
+    # where is backend running?
+    docker inspect $(docker-compose ps -q backend) | \
+       jq .[0].NetworkSettings.Networks.rsd_default.IPAddress
+    ```
 
-Run-time settings should be reachable through `[BASE_URL]/settings.json`, see e.g. `public/settings.json`.
-This settings file should be overriden in production.
+1.  `cd admin`
+1.  Install yarn, `npm install -g yarn` (Note: there exists a similarly named package, Apache Hadoop YARN --please ignore
+    that, it's something else)
+1.  Install the `admin` service's dependencies: `yarn install`
+1.  In `package.json`, replace (only) the `localhost` part of the URLs listed under `proxy` with the IP addresses for
+    `/api` using `backend`'s IP and for `auth` using `auth`'s IP; leave the port numbers as they were
+1.  Start the admin service in a development server: `yarn start`. It will tell you where to go to check the `admin`
```

Comment:
To login in to admin interface you need to set the callback to `http://localhost:8000/auth/get_jwt` on https://github.com/settings/developers